sdk/go/Makefile

```diff
@@ -38,7 +38,7 @@ test_fast:: $(TEST_ALL_DEPS)
 		$(GO_TEST_FAST) $(shell go list ./... | grep -v /vendor/ | grep -v templates)
 
 test_auto:: $(TEST_ALL_DEPS)
-	@$(GO_TEST) $(TEST_AUTO_PKGS)
+	@$(GO_TEST) -ldflags "-X github.com/pulumi/pulumi/sdk/v3/go/common/version.Version=${VERSION}" $(TEST_AUTO_PKGS)
```

Comment:
This is no good, if users want to run tests using the Automation API, they would need to do the same.